corehq/apps/reports/v2/reports/explore_case_data.py

```diff
@@ -53,11 +57,13 @@ class ExploreCaseDataReport(BaseReport):
             title=ugettext_lazy("Case Name"),
             name='case_name',
             width=200,
+            sort='asc',
         ),
         ColumnMeta(
             title=ugettext_lazy("Case Type"),
             name='@case_type',
             width=200,
+            sort=None,
```

Comment:
Why isn't sorting by case type allowed?

Reply:
this is just saying that sort is not activated on that column. not that it isn't allowed.